youtube_dl/extractor/googledrive.py

```diff
@@ -265,6 +265,8 @@ def add_source_format(src_url):
             subtitles_id = ttsurl.encode('utf-8').decode(
                 'unicode_escape').split('=')[-1]
 
+        self._downloader.cookiejar.clear('.google.com')
```

Comment:
With this you wipe **all cookies** for `.google.com` domain with corresponding consequences. You must only touch minimal relevant cookies for minimal relevant domain.

Reply:
You are entirely right! 
I've pushed a fix to be specific about which cookie to nuke.